src/ssl.c

```diff
@@ -34107,6 +34107,121 @@ int wolfSSL_DH_set0_key(WOLFSSL_DH *dh, WOLFSSL_BIGNUM *pub_key,
     return SetDhInternal(dh);
 }
 
+WOLFSSL_DH* wolfSSL_DH_get_2048_256(void)
```

Comment:
Where did these parameters come from?
(standard)

Reply:
RFC 5114

Reply:
Please put this into the code.
Thanks.

Reply:
Added a comment.